modules/kubernetes-addons/cert-manager/locals.tf

```diff
@@ -20,22 +20,13 @@ locals {
     var.helm_config
   )
 
-  set_values = [
-    {
-      name  = "serviceAccount.name"
-      value = local.service_account_name
-    },
-    {
-      name  = "serviceAccount.create"
-      value = false
-    }
-  ]
```

Comment:
First change the following 

from

```hcl
service_account_name = "${local.name}-sa"
```

to

```hcl
service_account_name = "cert-manager"
```

this block should stay with  `local.service_account_name`

Also, add a note here saying AWS PrivateCA is expecting the service account name as `cert-manager`. refer to docs if you can